File: sprint4/map.java


public class Main {

    // Create a HashMap object called capitalCities
    HashMap<String, String> capitalCities = new HashMap<String, String>();
    // Add keys and values (Country, City)
    capitalCities.put("qw", "London");
    capitalCities.put("Germany", "Berlin");
    capitalCities.put("Norway", "Oslo");
    capitalCities.put("USA", "Washington DC");
    capitalCities.remove("USA");
    capitalCities.clear();
    System.out.println(capitalCities);

}

File: sprint4/map.py
class Main:
    #  Create a HashMap object called capitalCities
    capitalCities = {}
    #  Add keys and values (Country, City)
    capitalCities["qw"] = "London"
    capitalCities["Germany"] = "Berlin"
    capitalCities["Norway"] = "Oslo"
    capitalCities["USA"] = "Washington DC"
    del capitalCities["USA"]
    capitalCities.clear()
    print(capitalCities)
